app/code/BVMyBlog/Blog/Setup/InstallSchema.php

```diff
@@ -0,0 +1,88 @@
+<?php
+declare(strict_types=1);
+namespace BVMyBlog\Blog\Setup;
+
+/**
+ * Class InstallSchema
+ *
+ * Setup InstallSchema
+ */
+class InstallSchema implements \Magento\Framework\Setup\InstallSchemaInterface
+{
+    /**
+     * Function install
+     *
+     * @param \Magento\Framework\Setup\SchemaSetupInterface $setup
+     * @param \Magento\Framework\Setup\ModuleContextInterface $context
+     * @throws \Zend_Db_Exception
+     */
+    public function install(
+        \Magento\Framework\Setup\SchemaSetupInterface $setup,
```

Comment:
Please **do not** use a Fully qualified class name (FQCN). Add all to **use** and use short forms.